roles/bootstrap-os/tasks/bootstrap-fedora.yml

```diff
@@ -40,6 +40,8 @@
 # See https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#managed-node-requirements
 - name: Install libselinux-python3
   package:
-    name: libselinux-python3
+    # name: libselinux-python3
+    name: python3-libselinux
```

Comment:
The package name `libselinux-python3` seems to be wrong with fedora 31 coreos (it is wrong with fedora 32 as well). It must be `python3-libselinux`. However, I have no idea when fedora changed the package name.

Reply:
I checked this on F30 and F31, dnf install libselinux-python3 installs python3-libselinux

Like:

dnf install libselinux-python3
Last metadata expiration check: 0:01:01 ago on Sun Apr 19 13:17:57 2020.
Dependencies resolved.
===========================================================================================
 Package                     Architecture    Version                Repository        Size
===========================================================================================
Installing:
 python3-libselinux          x86_64          2.9-3.1.fc30           updates          162 k

Transaction Summary
===========================================================================================
Install  1 Package

Total download size: 162 k
Installed size: 635 k
Is this ok [y/N]:
So good to use the canonical package name. 👍